spec/factories.rb

```diff
@@ -3,6 +3,7 @@
     title { Faker::Company.catch_phrase }
     description { Faker::Hacker.say_something_smart }
     cover_url { Faker::Avatar.image(nil, '100x200') }
+    published_at { Faker::Date.forward(23) }
```

Comment:
@Fatpaher Achtung! The book from the future! I think there will be better to use `Faker::Date.between` and some dates from the past. Like between 1 year ago and 1 day ago.